lib/web/static/js/app.js

```diff
@@ -0,0 +1,81 @@
+const error_alert_markup = `
+<div class="alert alert-danger alert-dismissible fade show" role="alert">
+  Error on the request to get more resources of this state.
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>`
+
+class API {
+  constructor() { this.url = "/" }
+
+  getResourcesForState(list, callback) {
+    let xhr = new XMLHttpRequest();
+    let state = list.dataset.state
+    page++;
+
+    xhr.open('GET', 'machinery/api/'+state+'/resources/'+page);
+    xhr.send(null);
+    xhr.onreadystatechange = function () {
+      const OK = 200;
+      const DONE = 4;
+
+      if (xhr.readyState === DONE) {
+        if (xhr.status === OK) {
+          callback(JSON.parse(xhr.response))
+        } else {
+          list.parentNode.querySelector('.info').innerHTML = error_alert_markup
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)